src/http-response-message.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Response object modeled after aurelia-http-client HTTPResponseMessage.
+ * @class
+ */
+export class HttpResponseMessage {
```

Comment:
Shouldn't this be AxiosHTTPResponseMessage?

Reply:
No, this mimics https://github.com/aurelia/http-client/blob/master/src/http-response-message.js for backwards compatibility.